interceptors/ratelimit/examples_test.go

```diff
@@ -2,32 +2,59 @@ package ratelimit_test
 
 import (
 	"context"
+	"fmt"
 
+	middleware "github.com/grpc-ecosystem/go-grpc-middleware/v2"
+	grpc_ratelimit "github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/ratelimit"
+	"github.com/juju/ratelimit"
 	"google.golang.org/grpc"
+)
 
-	middleware "github.com/grpc-ecosystem/go-grpc-middleware/v2"
-	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/ratelimit"
+// Rationale:
+// we use in the rate-limiter interceptor a "tocken-bucket" algorithm
```

Comment:
Rationale for what? Can we elaborate?

Also can we write comments full sentence?